va/va_drmcommon.h

```diff
@@ -155,4 +160,23 @@ typedef struct _VADRMPRIMESurfaceDescriptor {
 } VADRMPRIMESurfaceDescriptor;
 
 
+typedef struct _VADRMPRIMESurfaceDescriptorExt {
+    VADRMPRIMESurfaceDescriptor surface_desc;
+    /* memory location , local memory or system memory */
+    union{
+        struct{
+            /* whether the memory is allocated on local memory */
+            uint32_t local_memory :1;
+            /* whether the memory region is enabled */
+            uint32_t memory_region_enable: 1;
```

Comment:
What does it mean for a memory region to be enabled?  I can't find any concept like this in the i915 driver.

Reply:
if there are multiple memory chip, and has different control. it should be true.

Reply:
I am not sure that this flag is actually needed. If there is `memory_region_index` specified, does not that mean that "memory region is enabled?"